server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -105,9 +119,10 @@ public class RemoteClusterStateService implements Closeable {
 
     public static final String CLUSTER_STATE_PATH_TOKEN = "cluster-state";
     public static final String INDEX_PATH_TOKEN = "index";
+    public static final String GLOBAL_METADATA_PATH_TOKEN = "global-metadata";
     public static final String MANIFEST_PATH_TOKEN = "manifest";
     public static final String MANIFEST_FILE_PREFIX = "manifest";
-    public static final String INDEX_METADATA_FILE_PREFIX = "metadata";
+    public static final String METADATA_FILE_PREFIX = "metadata";
```

Comment:
nit: could have kept it `INDEX_METADATA_FILE_PREFIX`as it is used for index metadata